### PR TITLE
Add additional clickpipes step to explain Schema Registry setup

### PR DESCRIPTION
**Pull request by tpanetti**
Added an additional step to better explain both the setup process for the Schema Registry and it's functionality.

This closes https://github.com/ClickHouse/clickpipes-platform/issues/771

### CODE REVIEW

**Comment by robot-ch-test-poll3** [comment]
*This is an automated comment for commit 98824edc8bcfedf274e68f31fc045d1791150fd4 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/1525/98824edc8bcfedf274e68f31fc045d1791150fd4/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>✅ success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

**Comment by jkaflik** [comment]
@tpanetti we should update it with changes by @kschlarman https://github.com/ClickHouse/control-plane/pull/5007

File: docs/en/integrations/data-ingestion/clickpipes/index.md [hunk 1]
```diff
@@ -47,6 +47,11 @@ ClickPipes is currently accessible in private preview. You can join our waitlist
   :::note
   AWS MSK authentication currently only supports SCRAM-SHA-512 authentication, IAM authentication is coming soon
   :::
+
+4a. __Optional for Schema Registry__: Specify the complete URL of your Schema Registry server along with the precise RESTful path to the ID representing your preferred schema document. This schema will serve as the validation benchmark for all messages from your topic, consequently blocking any messages that fail the validation against the designated schema. Additionally, the Schema Registry allows new schema retreival via JSON_SR messages. In scenarios where your JSON_SR message carries a Schema Registry ID differing from the current one, Clickpipes will fetch the corresponding schema and use it for future validation.
```
Comment (kschlarman) [inline docs/en/integrations/data-ingestion/clickpipes/index.md:51]: Should we specify that currently, only JSON schema are supported?

Comment (kschlarman) [reply docs/en/integrations/data-ingestion/clickpipes/index.md:51]: Maybe we also call out that we don't support schema references either? At least they didn't work in my testing

**Comment by tpanetti** [comment]
> @tpanetti we should update it with changes by @kschlarman [ClickHouse/control-plane#5007](https://github.com/ClickHouse/control-plane/pull/5007)

Added

File: docs/en/integrations/data-ingestion/clickpipes/index.md (hunk 1 above)
Comment (tpanetti) [reply docs/en/integrations/data-ingestion/clickpipes/index.md:51]: Added a note for the supported JSON format and lack of support for Schema References.

**Review by kschlarman — APPROVED** [review]

**Pull request merged**